src/common/src/config.rs

```diff
@@ -20,6 +20,8 @@ use serde::{Deserialize, Serialize};
 use crate::error::ErrorCode::InternalError;
 use crate::error::{Result, RwError};
 
+pub const MAX_CONNECTION_WINDOW_SIZE: u32 = (1 << 31) - 1;
```

Comment:
```suggestion
pub const MAX_CONNECTION_WINDOW_SIZE: u32 = i32::MAX as _;
```

Reply:
it is from https://github.com/hyperium/h2/blob/756384f4cdd62bce3af7aa53a156ba2cc557b5ec/src/proto/mod.rs#L33
But seems I can't get it from tonic. So I just copied that.